plugins/inputs/zipkin/zipkin.go

```diff
@@ -22,6 +22,11 @@ import (
 //go:embed sample.conf
 var sampleConfig string
 
+var (
+	// defaultNetwork is the network to listen on; use only in tests.
+	defaultNetwork = "tcp"
+)
+
```

Comment:
If it's only used in tests, why not define it there?

Reply:
It is used both in prod and test. In test it is used to overwrite network.